src/gather/tests/test_commands.py

```diff
@@ -31,6 +31,23 @@ def main4(args):
 def weird_function():
     pass
 
+CONFLICTING_COMMANDS = gather.Collector()
```

Comment:
Venusian only registers module-level things. The theoretically correct thing to do here is to generate ad-hoc modules, but that seemed like an overkill. The sharing should work correctly.